CO-2464

Alaggaerion opened the pull request:
Added no index for controller routes in sms_sponsorship

Alaggaerion committed:
CO-2464

Added no index for controller routes in sms_sponsorship

ecino reviewed (changes requested):
Please change the file `oca_dependencies.txt` to change the repository `website` so that it takes the CompassionCH branch instead of the OCA one where the module `no_index` is not available.

Alaggaerion committed:
CO-2464
Updated oca_dependencies.txt to take compassionCH/website module instead of the oca one
